### DISCUSSION TITLE
Location wrong

### DISCUSSION THREAD

**Discussion by zhuchangyi** [community/community#37778] [GitHub Education] [answered]
I can’t get the edu pack for location reason, i am at school right now but the processing always turned that i am not near any campus, i also turn on my browser location and i all failed.
![Uploading E887C9DB-D421-42E3-BC3D-46D5FD9E0421.jpeg…]()

**Comment by samus-aran** [comment, marked as answer]
Hi @zhuchangyi 👋 

Thank you for raising this. We've raised this with the internal Education team and are currently investigating the mismatch. When there is an update we'll post here in the discussion. We thank you for your patience and again for reaching out to us 👍

**Comment by Bouba-hrg** [comment]
I also had the same issue and they don't give solution to solve the problem

**Comment by Mejiabrayan** [comment]
Hi, I've also came across this issue. I'm currently attending a coding bootcamp which should be fine since it's technically online, however, I'm visiting family for a month abroad and unable to verify because I'm not in the proper location. 

This frustrating because it interferes with my course work.

**Reply by samus-aran** [reply]
Hi @Mejiabrayan 👋 

First off I want to thank you for sharing your experiences and that we absolutely acknowledge your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and institutes around the world and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient with it.

What does your rejection reasons say (a screenshot would help) and we can take it from there. We do support remote learning but your evidence should be able to confirm this and is eligible (the photo image should be readable even for a human).

**Comment by juliaramosguedes** [comment]
I'm also facing the same issue. I'm doing a bachelor's degree, and I can't apply for the benefits because of this error.

**Comment by tinnnkie** [comment]
I am dealing with the same error. Every time I try to apply, fill in the form and press continue, I am back to the same page. 
https://education.github.com/discount_requests/pack_application

Tried different browsers, but seems to me the location is the problem because when I delete cookies or try a different browser asks me about the location only the first time, and then redirects me back with no further location detection after I try to reapply again.
